redhat-cop/businessautomation-cop
# error handling example

#### What is this PR About?
a kjar demonstrating the error handling strategy

#### How do we test this?
- import in Business Central and deploy
- launch the main process
- see the example in https://youtu.be/RPbM2B8TAo8
 
cc: @redhat-cop/businessautomation-cop
